apps/server/src/schemas/template.schema.ts

```diff
@@ -0,0 +1,18 @@
+import { Prop, Schema, SchemaFactory } from '@nestjs/mongoose';
+import { HydratedDocument } from 'mongoose';
+
+export type TemplateDocument = HydratedDocument<Template>;
+
+@Schema()
+export class Template {
```

Comment:
Actually the `Template` model is that below, I think we need update this schema.

`{
    "id": "string",
    "name": "string",
    "subject": "string",
    "html": "string"
  }`